eng/pipelines/templates/steps/sync-repo-branch.yml

```diff
@@ -1,17 +1,102 @@
 parameters:
-  Repos: # Provde a list of repos
+  # Provde a list of repos
+  # Example:
+  #   Azure/azure-rest-api-specs:
+  #     Branch: master
+  #     TargetRepos: 
+  #       Azure/azure-rest-api-specs-pr:
+  #       azure-sdk/azure-rest-api-specs:
+  #         Branch: dev
+  #         Rebase: true
+  #   Azure/azure-sdk-for-go:
+  #     Branch: master
+  #     TargetRepos: 
+  #       Azure/azure-sdk-for-go-pr:
+  - name: Repos
+    type: object
+  # Github PAT
+  - name: GH_TOKEN
+    type: string
+  # Github alias for git config
+  - name: GithubAlias
+    type: string
+    default: azure-sdk
+
 
 steps:
 
+- pwsh: |
+    git config --global user.email "${{ parameters.GithubAlias }}@microsoft.com"
```

Comment:
We should avoid changing the config on the system. Instead we should just pass this info to any git command that needs it similar to https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/git-branch-push.ps1#L61. That way we don't accidently pollute the git config if we use this in conjunction with other pipeline stuff.

Reply:
Resolved